debian/logitechmediaserver.default

```diff
@@ -1,4 +1,10 @@
-# User to run Logitech Media Server as
+# User to run Logitech Media Server as.
+# Only applicable to platforms that do not use systemd.
+# Running the server as a user other than the default squeezeboxserver
+# is not recommended.
```

Comment:
How about a brief note explaining how to change the user with systemd 
```
On platforms running systemd this will be ignored. 
If you want to change the user LMS runs as you need to use a 
systemd override file. 
With recent versions of systemd you can simply do this:
- sudo systemctl edit logitechmediaserver
- in that file add the line 
User=<new username>

Reply:
Possibly.

Bear in mind that the Debian installer sets everything up to run under user squeezeboxserver.

I would not personally be inclined to encourage users to depart from that. Knowledgeable users can figure it out for themselves.

That's my take, anyway. But I'm more than happy to change if that's what's wanted.